src/game.cpp

```diff
@@ -1211,11 +1211,13 @@ ReturnValue Game::internalMoveItem(Cylinder* fromCylinder, Cylinder* toCylinder,
 		return retMaxCount;
 	}
 
-	if (moveItem->getDuration() > 0) {
-		if (moveItem->getDecaying() != DECAYING_TRUE) {
-			moveItem->incrementReferenceCounter();
-			moveItem->setDecaying(DECAYING_TRUE);
-			g_game.toDecayItems.push_front(moveItem);
+	if (moveItem) {
+		if (moveItem->getDuration() > 0) {
```

Comment:
I would have preferred `if (moveItem && moveItem->getDuration() > 0) {` to reduce nesting.

Reply:
sure, will do that